users/users-db.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../data/dbConfig")
+
+module.exports = {
+  addUser,
+  findById,
+  findBy,
+  getAllUsers
+}
+
+async function addUser(user) {
+  const [id] = await db("users").insert(user)
+
+  return findById(id)
+}
```

Comment:
I don't see the need to seek the inserted user and return their data. If you return a valid id then you can assume the user was successfully added. If you needed the data, you should already have it where you use this addUser function.